src/server/bg_services/server_monitor.js

```diff
@@ -69,61 +66,11 @@ async function run_monitors() {
 }
 
 function _check_dns_and_phonehome() {
-    dbg.log2('_check_dns_and_phonehome');
-    return phone_home_utils.verify_connection_to_phonehome()
-        .then(res => {
-            switch (res) {
-                case "CONNECTED":
-                    monitoring_status.dns_status = "OPERATIONAL";
-                    monitoring_status.ph_status = {
-                        status: "OPERATIONAL",
-                        test_time: moment().unix()
-                    };
-                    break;
-                case "MALFORMED_RESPONSE":
-                    monitoring_status.dns_status = "OPERATIONAL";
-                    monitoring_status.ph_status = {
-                        status: "FAULTY",
-                        test_time: moment().unix()
-                    };
-                    break;
-                case "CANNOT_CONNECT_PHONEHOME_SERVER":
-                    monitoring_status.dns_status = "OPERATIONAL";
-                    monitoring_status.ph_status = {
-                        status: "UNREACHABLE",
-                        test_time: moment().unix()
-                    };
-                    Dispatcher.instance().alert('MAJOR',
-                        system_store.data.systems[0]._id,
-                        `Phone home server could not be reached`,
-                        Dispatcher.rules.once_daily);
-                    break;
-                case "CANNOT_CONNECT_INTERNET":
-                    monitoring_status.internet_connectivity = "FAULTY";
-                    Dispatcher.instance().alert('MAJOR',
-                        system_store.data.systems[0]._id,
-                        `Phone home server could not be reached, phone home connectivity is used for proactive
-                         support product statistics analysis. Check internet connection`,
-                        Dispatcher.rules.once_daily);
-                    break;
-                case "CANNOT_RESOLVE_PHONEHOME_NAME":
-                    monitoring_status.dns_status = "FAULTY";
-                    Dispatcher.instance().alert('MAJOR',
-                        system_store.data.systems[0]._id,
-                        `DNS server/s cannot resolve Phone home server name`,
-                        Dispatcher.rules.once_daily);
-                    break;
-                case "CANNOT_REACH_DNS_SERVER":
-                    monitoring_status.dns_status = "UNREACHABLE";
-                    break;
-                default:
-                    break;
-            }
-            if (_.isEmpty(server_conf.dns_servers)) {
```

Comment:
Notice that we delete the dns_servers in case we do not have them in our server_conf.
This change was removed in your new proposed code, this means that we will always send `dns_status = "OPERATIONAL"` even when we do not have `dns_servers` in our `server_conf`.
Currently, this value is exported in cluster_info which is used by the FE (but not this particular value from what I saw).

Reply:
fixed